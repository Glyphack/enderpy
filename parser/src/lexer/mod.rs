mod diagnostics;
pub mod fstring;
pub mod lexer;
