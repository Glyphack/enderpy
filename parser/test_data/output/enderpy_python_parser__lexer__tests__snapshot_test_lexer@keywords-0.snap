---
source: parser/src/lexer/mod.rs
description: False None True and as assert async await
---
0,5: False
6,10: None
11,15: True
16,19: And
20,22: As
23,29: Assert
30,35: Async
36,41: Await
